### DISCUSSION TITLE
Full Screen Aspect ratio logic

### DISCUSSION THREAD

**Discussion by fahad86** [TeamNewPipe/NewPipe#11580] [Questions] [answered]
If the user is in portrait mode and the video aspect ratio is 1:1 then the full Screen button should open full screen in portrait (current) orientation.

**Comment by ShareASmile** [comment, marked as answer]
duplicate of #10595.

Please Note "Discussion is for questions, for bugs is the issue tracker".

Use that next time & do a Search in issue tracker with relevant  terms/keywords like in your case full screen button & you will find relevant  tickets. If nothing relevant is found you can create a new issue. Thank you.
